.devcontainer/devcontainer.json

```diff
@@ -34,7 +34,7 @@
 	// "forwardPorts": [],
 
 	// Uncomment the next line to run commands after the container is created - for example installing curl.
-	"postCreateCommand": "container/database.sh && container/library.sh && ldconfig",
+	"postCreateCommand": "/workspace/scripts/initialize_dev_environment.sh",
 	
```

Comment:
Attempt to use `initializationCommand` (https://containers.dev/implementors/json_reference/) to copy a `.devcontainer/devcontainer.env` file to configuration/.env if an env file does not already exist.